recipes.js

```diff
@@ -9,3 +9,78 @@ mongoose.connect('mongodb://localhost/recipeApp')
     console.error('Error connecting to mongo', err)
   });
 
+
+  
+  
+  const recipeSchema = new Schema ({
+    title: {
+      type: String,
+      required: true
+    },
+    level: {
+      type: String,
+      enum: ['Easy Peasy', 'Amateur Chef', 'UltraPro Chef'],
+    },
+    ingredient: Array, 
+    cousine: {
+      type: String,
+      required: true,
+    },
+    dishType: {
+      type: String,
+      enum: ['Breakfast', 'Dish','Snack','Drink','Dessert','Other'],
+    },
+    image: {
+      type: String,
+      default: 'https://images.media-allrecipes.com/images/75131.jpg',
+    },
+    duration: {
+      type: Number,
+      min: 0,
+    },
+    creator: String,
+    create: {
+      type: Date,
+      default: Date.now,
+    }
+    
+  })
+    // We create a document in the database "recipe"
+  
+  const Recipe = mongoose.model("Recipe", recipeSchema);
+  
+  // Each time we run the app, this will create a field in the database. Create a field in the recipe database. In brackets the schema.
+    // Recipe.insertMany(data)
+    // .then (recipeDoc =>{
+    // console.log("Create SUCCESSS", recipeDoc);
+    // })
+    // .catch (err => {
+    // console.log("Create FAILURE!!", err);
+    // });
+
+    // Recipe.findOneAndUpdate(
+    //     {title: {$eq: "Rigatoni alla Genovese"}}, 
```

Comment:
It works fine that way but you don't really need the $eq operator here, it would also work with 
```
 {title: "Rigatoni alla Genovese"},
```